pkgs/dart_mcp_server/README.md

```diff
@@ -100,11 +100,20 @@ For more information, see the official Cursor documentation for
 [installing MCP servers](https://docs.cursor.com/context/model-context-protocol#installing-mcp-servers).
 
 ### Visual Studio Code Copilot
-<!-- TODO: update these docs to mention the automatic configuration provided by the Dart
-VS Code extension. -->
 
-To configure the Dart MCP server for a single project, edit the `.vscode/mcp.json`
-file in your workspace:
+To configure the Dart MCP server for a single project, edit your VS Code user settings to
```

Comment:
nit: Adding it to your User Settings will enable it for all workspaces. If you add it to _Workspace Settings_ then it will be enabled only for that workspace. "project" is probably the wrong term to use here because it's always enabled (or not) for an open workspace (which may be a single folder) and is never specifically scoped to a single project (unless of course your workspace just happens to be a single project).